packages/spec/src/spec/marks/Density.ts

```diff
@@ -44,6 +44,12 @@ export interface Density1DOptions {
    * Defaults to 1024.
    */
   bins?: number | ParamRef;
+
+  /**
+   * Flag indicating if density estimates should be normalized by dividing
+   * by the total point mass. Defaults to false.
+   */
+  normalize?: boolean | ParamRef;
```

Comment:
I wonder whether we should make this an enum so that we can also support normalization by the max instead of the sum for example. 

```suggestion
  normalize?: 'sum' | 'max' | ParamRef;
```

Reply:
I’m ok doing this. I wonder if we should still support `true` as an alias for `”sum”`.

More generally `max` could lead to misinterpretation, but FWIW ggplot and other tools also support it. (For example to make violin plots where each violin has the same width.)

Reply:
I've updated the PR to include `max` normalization. The difference is importantly subtle: `sum` is applied using the unsmoothed grid (total point mass, as smoothing may push mass outside the visualized domain), while `max` is applied relative to the smoothed values.